moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1112,9 +1111,8 @@ Eigen::MatrixXd RobotState::getJacobian(const JointModelGroup* group,
 {
   Eigen::MatrixXd result;
   if (group == nullptr)
-  {
     throw Exception("Unable to compute Jacobian");
-  }
+
```

Comment:
Even if the parentheses were removed, it was confirmed that the code had the same execution result.